ush/run_fv3jedi_exe.py

```diff
@@ -9,7 +9,7 @@
 import yaml
 
 
-def run_atm_var_analysis(yamlconfig):
+def run_fv3jedi_exe(yamlconfig):
```

Comment:
Nitpicky comment (feel free to ignore).   If we envision supporting more than `fv3jedi_*` executables, we could drop `fv3` and use `run_jedi_exe`.   The script would then be renamed `run_jedi_exe.py`

Reply:
no you're right, and @guillaumevernieres would appreciate this :-)

Will fix in a minute.

Reply:
Fix looks good.   Thanks!